### DISCUSSION TITLE
Updater window ignores "skip this release"...

### DISCUSSION THREAD

**Discussion by d-mac-648** [CommandPost/CommandPost#3264] [Technical Support]
The updater window is popping up repeatedly (more than once per day).

Choosing "skip this release" seems to be ignored.

Thanks for looking into this.

**Comment by latenitefilms** [comment]
Will investigate. Have you tried unchecking "Check for updates" in General Preferences in the meantime?

<img width="1000" alt="image" src="https://github.com/CommandPost/CommandPost/assets/22286696/54628dfc-d0c4-4c9e-bf0b-20a43d7c9735">

**Comment by d-mac-648** [comment]
Chris,

Thanks for the prompt response.

Yes, thanks for suggesting turning off update checking!


Cheers,

Dave
